packages/create-react-app/index.js

```diff
@@ -173,10 +173,14 @@ function install(packageToInstall, verbose, callback) {
 function run(root, appName, version, verbose, originalDirectory, template) {
   var packageToInstall = getInstallPackage(version);
   var packageName = getPackageName(packageToInstall);
-
   install(packageToInstall, verbose, function(code, command, args) {
     if (code !== 0) {
-      console.error(chalk.cyan(command + ' ' + args.join(' ')) + ' failed');
+      console.log();
+      console.error('Aborting installation.', chalk.cyan(command + ' ' + args.join(' ')), 'has failed.');
+      console.log();
+      console.log('Deleting', chalk.cyan(appName), 'from', chalk.cyan(originalDirectory));
+      fs.removeSync(path.join(originalDirectory, appName));
```

Comment:
> There are cases when we allow installing into an _existing_ directory

Is there a switch to force that? I tested it and it wouldn't allow me even if the folder contained unrelated files to the project.

```sh
foosbar/
└── unrelatedfile.txt

0 directories, 1 file
```

```
The directory foosbar contains files that could conflict.
Try using a new directory name.
```